lib/src/signed_video_internal.h

```diff
@@ -220,6 +229,8 @@ struct _gop_info_t {
   // when SV_AUTHENTICITY_LEVEL_FRAME.
   uint8_t tmp_hash[MAX_HASH_SIZE];  // Memory for storing a temporary hash needed when a NALU is
   // split in parts.
+  linked_hash_t linked_hash_data;  // Stores linked hash data for liked hash method.
```

Comment:
If we should keep the same structure as the other hashes in `gop_info_t` this should not be put in a special struct but in a consecutive memory slot. And also rename because "hash data" in general cases means the data that is hashed.
```
uint8_t linked_hashes[2 * MAX_HASH_SIZE];
```

Reply:
And since you do not use the `linked_hash_t` struct anymore, you can remove it.